Change swiftdialog to notification (from dialog)

pmex opened the pull request:
It's less intrusive for users with notifications

erikstam reviewed (commented):
In my opinion we should have a choice between --mini and --notification. What do you think?

pmex commented:
I was actually considering that when I created this PR. If I can come up with a good way to implement it without breaking anything, I'll add a commit or two to this PR.

pmex commented:
I've added the possibility to choose notification type with Swift Dialog.

Default is reverted a mini dialog, but changing `swiftDialogNotification` to `notification` will use notifications instead.

Also added `FallBacktoAS=true` to swiftdialog, since it was missing

N.B, if you also merge https://github.com/erikstam/uninstaller/pull/25 it will be possible to set that argument differently for each label.
